IPython/html/static/widgets/js/widget_bool.js

```diff
@@ -129,10 +129,11 @@ define([
 
                 var description = this.model.get('description');
                 this.$el.attr("title", this.model.get("tooltip"));
-                if (description.trim().length === 0) {
+                this.$el.text(description);
+                var icon = this.model.get("icon");
```

Comment:
Is there any chance that `icon` will be undefined or null instead of a string? If so, that may need to be handled below, where it's assumed that `icon` is always a string.